backend/src/services/encounter.service.ts

```diff
@@ -12,9 +12,21 @@ const updateEncounter = async (objectID: string, encounterDetails: EncounterMode
     return updatedEncounter;
 }
 
+const deleteEncounterPerson = async (personID: string) => {
+  // Service is used for deletePersons endpoint
+
+  await Encounter.updateMany({ }, { $pullAll: {persons: [{ _id: personID}]} });
+  const empty_encounters = await Encounter.find({persons : {$exists:true, $size:0}});
+  await Encounter.deleteMany({persons: {$exists: true, $size: 0}});
+
```

Comment:
Might be nice to have a few comments for this part such as:
* Remove the person with the supplied ID from all existing encounters
* Store references to the encounters that no longer contain persons so they can be removed from Users later
* Delete all the empty encounters